docs/source/markdown/podman-run.1.md

```diff
@@ -495,7 +495,7 @@ Tune a container's memory swappiness behavior. Accepts an integer between *0* an
 
 Attach a filesystem mount to the container
 
-Current supported mount TYPEs are **bind**, **volume**, and **tmpfs**.
+Current supported mount TYPEs are **bind**, **volume**, and **tmpfs**. <sup>[[1]](#Footnote1)</sup>
```

Comment:
Why is there a footnote here?

Reply:
right after this line, there are examples that mention bind propagation types. they're in a block quote so it won't take the footnote markdown, so i thought the next best place was to put it here. let me know if i should move it or put it somewhere else

Reply:
Ok